### ISSUE TITLE
want to create bullet chart

### ISSUE THREAD

**Issue by mkhajuriwala** [google/google-visualization-issues#2254]
i want to create bullet chart. Can i create one using google

**Comment by jcw-** [comment]
I don't believe there is one natively available. You could try raising this question on the [forums ](https://groups.google.com/forum/#!forum/google-visualization-api)though. Also, check out related question  #1858

**Comment by manoharreddyporeddy** [comment]
Yes, bullet charts are one of the two things that are missing in Google Charts https://en.wikipedia.org/wiki/Comparison_of_JavaScript_charting_libraries
